app/templates/partials/summary/checkbox.html

```diff
@@ -1,18 +1,34 @@
 {% set not_answered = 'No answer provided' %}
-<ul class="u-m-no">
-      {% for selected_answer in answer.value %}
-        <li>{{selected_answer.label}}
-            {%- if answer.child_answer_value is not none and selected_answer.should_display_other %}
-            <ul>
-                <li>
-                    {%- if answer.child_answer_value == ''  -%}
-                        {{not_answered}}
-                    {%- else -%}
-                        {{ answer.child_answer_value }}
-                    {%- endif -%}
-                </li>
-            </ul>
+{%- if answer.value|length > 1 -%}
+    <ul class="u-m-no">
+          {% for selected_answer in answer.value %}
+            <li>{{selected_answer.label}}
+                {%- if answer.child_answer_value is not none and selected_answer.should_display_other %}
+                <ul>
+                    <li>
+                        {%- if answer.child_answer_value == ''  -%}
+                            {{not_answered}}
+                        {%- else -%}
+                            {{ answer.child_answer_value }}
+                        {%- endif -%}
+                    </li>
+                </ul>
+                {%- endif -%}
+            </li>
+          {% endfor %}
+    </ul>
+{%- else -%}
+    {% set selected_answer = answer.value[0] %}
+    {{selected_answer.label}}
```

Comment:
This block seems to be duplicated? can we set this as a block and reuse?

Reply:
i tried fiddling with this yday, and could't get it to work perfectly, unless we use a filter but yeah i agree if possible we should do.

Reply:
Couldn't set this as a block as the duplicated section contains a local variable.  I'm not sure a filter is quite the right tool either as it's mostly display logic.  I've factored it out into a macro.